### PR TITLE
Fix/4749 segmented control on 12.5

### PR DESCRIPTION
**Pull request by KaiTeuber**
## Description
Added some color and formatting options to make UISegmented Control look a bit like the iOS 13 Designs

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-4749

## Screenshots (iOS 12.5 doesn't support dark mode)
Screen Shot 2021-01-26 at 16 11 16

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Day/DiaryDayViewController.swift [hunk 1]
```diff
@@ -100,11 +100,30 @@ class DiaryDayViewController: UIViewController, UITableViewDataSource, UITableVi
...
-		segmentedControl.setTitleTextAttributes([NSAttributedString.Key.font: UIFont.enaFont(for: .subheadline)], for: .normal)
-		segmentedControl.setTitleTextAttributes([NSAttributedString.Key.font: UIFont.enaFont(for: .subheadline, weight: .bold)], for: .selected)
-
 		segmentedControl.setTitle(AppStrings.ContactDiary.Day.contactPersonsSegment, forSegmentAt: 0)
 		segmentedControl.setTitle(AppStrings.ContactDiary.Day.locationsSegment, forSegmentAt: 1)
+
+		// required to make segement control look a bit like iOS 13
+		if #available(iOS 13, *) {
+		} else {
+			Log.debug("setup segmented control for iOS 12")
```
Comment (marcussc) [inline src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Day/DiaryDayViewController.swift:109]:
```suggestion
			Log.debug("setup segmented control for iOS 12", log: .ui)
```

**Review by marcussc — APPROVED** [review]

**Review by pascal-brause — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/AppDelegate.swift [hunk 2]
```diff
@@ -439,6 +439,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate, CoronaWarnAppDelegate, Re
 			NSAttributedString.Key.font: UIFont.preferredFont(forTextStyle: .largeTitle).scaledFont(size: 28, weight: .bold),
 			NSAttributedString.Key.foregroundColor: UIColor.enaColor(for: .textPrimary1)
 		]
+
+		appearance.shadowImage = UIImage()
```
Comment (nickguendling) [inline src/xcode/ENA/ENA/Source/AppDelegate.swift:443]: Do we want this? Does this change anything on iOS 123 and up?

**Review by nickguendling — CHANGES_REQUESTED** [review]

File: src/xcode/ENA/ENA/Source/AppDelegate.swift (hunk 2 above)
Comment (nickguendling) [reply src/xcode/ENA/ENA/Source/AppDelegate.swift:443]: Also: maybe we can add some spacing above the segmented control, we have also done this for iOS 12 on other screens, see [here](https://github.com/corona-warn-app/cwa-app-ios/pull/1828)

Comment (KaiTeuber) [reply src/xcode/ENA/ENA/Source/AppDelegate.swift:443]: Spacing is already done - the missing shadow doesn't change anything on iOS 13

Comment (KaiTeuber) [reply src/xcode/ENA/ENA/Source/AppDelegate.swift:443]: > Do we want this? Does this change anything on iOS 13 and up?

iOS 12 has a drop shadow by default, iOS 13+ don't have one. This didn't changed anything in my tests

**Review by nickguendling — APPROVED** [review]

**Pull request merged**